### DISCUSSION TITLE
List items getting double marker and before bullet points

### DISCUSSION THREAD

**Discussion by junctionapps** [tailwindlabs/tailwindcss#8037] [Help]
Using Wagtail CMS, and just upgraded from Tailwind CSS 2.xx to 3.0.23 and now noticing a double bullet point on list items (about 70% certain it was a single bullet point before the upgrade). As a heads up, the text coming from CMS is all wrapped in `prose`.  I'm trying to determine the right approach to getting just one bullet point without breaking prose setup too badly but I'm not sure if I should overwrite the `marker` or the `before` (theme's typography in config.js?) or if I've got some other residual causing this. 

![image](https://user-images.githubusercontent.com/8778630/161452396-b07fc621-cc57-4694-84c0-e9a226bb8e3a.png)

Page live at: 
https://banookcanoeclub.com/membership-registration/#banook-benefits

**Comment by peterjthomson** [comment]
Where did you get to with this? I'm struggling with Prose adding its own style of bullet points next to any ones that are manually specified in the input html.

**Reply by junctionapps** [reply]
I haven't addressed it yet. Still showing as double bullet points at https://banookcanoeclub.com/membership-registration/#banook-benefits 

Not a critical issue, and I'll get to it, likely by attempting to remove the marker (the first point point) as it is less centered on the line.
![image](https://user-images.githubusercontent.com/8778630/170713648-b3181746-1e1c-4d6d-91e1-4dc387778044.png)

**Reply by junctionapps** [reply]
@peterthomson as a side note: I solved this by removing a long forgotten link to "https://unpkg.com/@tailwindcss/typography@0.2.x/dist/typography.min.css" that was being loaded into a few of my base level templates (using Wagtail). It was residual from when I first started trying out the `prose` class, but before I configured TailwindCSS correctly. It got caught when I was doing an upgrade.
